RELEASING.md

```diff
@@ -59,12 +60,24 @@ Perform these steps in PR branches based on release-next (trunk).
 )
 ```
 
-## Release Pre-requisites
+## Pre-Release
 
-Perform these steps in the release-next (trunk) branch which is based on main to release Ziti.
+Perform these steps in "release-next" (the default branch based on "main") to create a pre-release Ziti.
 
-1. Create a PR to merge release-next to main. Release happens by merging from the release-next branch to main.
-2. Ensure PR checks succeed.
+1. Create a PR to merge "release-next" to "main."
+1. Ensure "main" checks succeed. Downstreams will not be released if any checks fail on same revision where a release is created.
+1. Push a tag like v*, typically on default branch HEAD to trigger the pre-release workflow named `release.yml`.
+
+## Stable and Latest Release
+
+Pre-releases are releases, but they're not promoted as "latest" in GitHub or automatically shipped downstream. Marking a
+release as not a prerelease makes it a stable release. There can be one stable release that's also marked "latest"
+(`isLatest: true`).
+
+1. After an arbitrary burn-in period, unmark "prerelease" in GitHub Releases (`isPrerelease: false`) to stop
+   discouraging users from consuming the now-stable release. This will automatically promote and advertise the downstreams.
```

Comment:
is this supposed to be `non-stable`?

`stop discouraging` reads like double negative, I had to think what it means